<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 127);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);
color: rgb(255, 255, 255);
font: 75 28pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="margin">
     <number>20</number>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonC">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 75 28pt &quot;Comic Sans MS&quot;;
color: rgb(255, 255, 255);
border-radius: 30px;</string>
    </property>
    <property name="text">
     <string>С</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbuttonsquare">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 75 28pt &quot;Comic Sans MS&quot;;
color: rgb(255, 255, 255);
border-radius: 30px;</string>
    </property>
    <property name="text">
     <string>√</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbutton9">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 0, 0);
	font: 75 28pt &quot;Comic Sans MS&quot;;
	color: rgb(255, 255, 255);
	border-radius: 30px; }

QPushButton:hover {	
	background-color: rgb(170, 170, 255);
	font: 75 28pt &quot;Comic Sans MS&quot;;
	color: rgb(255, 255, 255);
	border-radius: 30px; }

QPushButton:pressed {
	background-color: rgb(110, 112, 255);
	font: 75 28pt &quot;Comic Sans MS&quot;;
	color: rgb(255, 255, 255);
	border-radius: 30px; }</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbutton6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 0, 0);
	font: 75 28pt &quot;Comic Sans MS&quot;;
	color: rgb(255, 255, 255);
	border-radius: 30px; }

QPushButton:hover {	
	background-color: rgb(170, 170, 255);
	font: 75 28pt &quot;Comic Sans MS&quot;;
	color: rgb(255, 255, 255);
	border-radius: 30px; }

QPushButton:pressed {
	background-color: rgb(110, 112, 255);
	font: 75 28pt &quot;Comic Sans MS&quot;;
	color: rgb(255, 255, 255);
	border-radius: 30px; }</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbuttondivide">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 75 28pt &quot;Comic Sans MS&quot;;
color: rgb(255, 255, 255);
border-radius: 30px;</string>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbuttonbackspace">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 75 28pt &quot;Comic Sans MS&quot;;
color: rgb(255, 255, 255);
border-radius: 30px;</string>
    </property>
    <property name="text">
     <string>⌦</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbutton4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>270</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 0, 0);
	font: 75 28pt &quot;Comic Sans MS&quot;;
	color: rgb(255, 255, 255);
	border-radius: 30px; }

QPushButton:hover {	
	background-color: rgb(170, 170, 255);
	font: 75 28pt &quot;Comic Sans MS&quot;;
	color: rgb(255, 255, 255);
	border-radius: 30px; }

QPushButton:pressed {
	background-color: rgb(110, 112, 255);
	font: 75 28pt &quot;Comic Sans MS&quot;;
	color: rgb(255, 255, 255);
	border-radius: 30px; }</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbutton1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 0, 0);
	font: 75 28pt &quot;Comic Sans MS&quot;;
	color: rgb(255, 255, 255);
	border-radius: 30px; }

QPushButton:hover {	
	background-color: rgb(170, 170, 255);
	font: 75 28pt &quot;Comic Sans MS&quot;;
	color: rgb(255, 255, 255);
	border-radius: 30px; }

QPushButton:pressed {
	background-color: rgb(110, 112, 255);
	font: 75 28pt &quot;Comic Sans MS&quot;;
	color: rgb(255, 255, 255);
	border-radius: 30px; }</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbutton5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 0, 0);
	font: 75 28pt &quot;Comic Sans MS&quot;;
	color: rgb(255, 255, 255);
	border-radius: 30px; }

QPushButton:hover {	
	background-color: rgb(170, 170, 255);
	font: 75 28pt &quot;Comic Sans MS&quot;;
	color: rgb(255, 255, 255);
	border-radius: 30px; }

QPushButton:pressed {
	background-color: rgb(110, 112, 255);
	font: 75 28pt &quot;Comic Sans MS&quot;;
	color: rgb(255, 255, 255);
	border-radius: 30px; }</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbutton7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 0, 0);
	font: 75 28pt &quot;Comic Sans MS&quot;;
	color: rgb(255, 255, 255);
	border-radius: 30px; }

QPushButton:hover {	
	background-color: rgb(170, 170, 255);
	font: 75 28pt &quot;Comic Sans MS&quot;;
	color: rgb(255, 255, 255);
	border-radius: 30px; }

QPushButton:pressed {
	background-color: rgb(110, 112, 255);
	font: 75 28pt &quot;Comic Sans MS&quot;;
	color: rgb(255, 255, 255);
	border-radius: 30px; }
</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbutton8">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 0, 0);
	font: 75 28pt &quot;Comic Sans MS&quot;;
	color: rgb(255, 255, 255);
	border-radius: 30px; }

QPushButton:hover {	
	background-color: rgb(170, 170, 255);
	font: 75 28pt &quot;Comic Sans MS&quot;;
	color: rgb(255, 255, 255);
	border-radius: 30px; }

QPushButton:pressed {
	background-color: rgb(110, 112, 255);
	font: 75 28pt &quot;Comic Sans MS&quot;;
	color: rgb(255, 255, 255);
	border-radius: 30px; }</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbutton2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>340</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 0, 0);
	font: 75 28pt &quot;Comic Sans MS&quot;;
	color: rgb(255, 255, 255);
	border-radius: 30px; }

QPushButton:hover {	
	background-color: rgb(170, 170, 255);
	font: 75 28pt &quot;Comic Sans MS&quot;;
	color: rgb(255, 255, 255);
	border-radius: 30px; }

QPushButton:pressed {
	background-color: rgb(110, 112, 255);
	font: 75 28pt &quot;Comic Sans MS&quot;;
	color: rgb(255, 255, 255);
	border-radius: 30px; }</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbuttonplus">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>340</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 75 28pt &quot;Comic Sans MS&quot;;
color: rgb(255, 255, 255);
border-radius: 30px;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbuttonminus">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>270</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 75 28pt &quot;Comic Sans MS&quot;;
color: rgb(255, 255, 255);
border-radius: 30px;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbuttonmulti">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>200</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 75 28pt &quot;Comic Sans MS&quot;;
color: rgb(255, 255, 255);
border-radius: 30px;</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbutton3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>340</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 0, 0);
	font: 75 28pt &quot;Comic Sans MS&quot;;
	color: rgb(255, 255, 255);
	border-radius: 30px; }

QPushButton:hover {	
	background-color: rgb(170, 170, 255);
	font: 75 28pt &quot;Comic Sans MS&quot;;
	color: rgb(255, 255, 255);
	border-radius: 30px; }

QPushButton:pressed {
	background-color: rgb(110, 112, 255);
	font: 75 28pt &quot;Comic Sans MS&quot;;
	color: rgb(255, 255, 255);
	border-radius: 30px; }</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbuttonequal">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>410</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 75 28pt &quot;Comic Sans MS&quot;;
color: rgb(255, 255, 255);
border-radius: 30px;</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbuttoncomma">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>410</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 75 28pt &quot;Comic Sans MS&quot;;
color: rgb(255, 255, 255);
border-radius: 30px;</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbutton0">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 75 28pt &quot;Comic Sans MS&quot;;
color: rgb(255, 255, 255);
border-radius: 30px;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>402</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
